Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
OK, if that works with OpenJDK I think we should go with Alpine + OpenJDK : it's very light, with no trademark/license issue